### ISSUE TITLE
error in extension in importing csv files 

### ISSUE THREAD

**Issue by anmolnehru** [JiahaoShan/Glacier-tax-1099-B-Stock-Transactions-Helper#3]
exported my rbnhd 1099 using your csv tool, the csv has headers formatted as expected, but the extension throws this error.

Choose File
ERROR: only csv with header 'name,acquired,sold,proceeds,cost' is supported!

I have used the tool at https://jiahaoshan.github.io/Robinhood-1099-B-Transactions-Export-Tool/

I can look at the csv and it does have the exepected accurate headers.

**Comment by JiahaoShan** [comment]
Hi, thanks for the feedback. I hope that the tools have at least helped you a bit. Could you help me figure this problem out? 

"you may simply right click on the Glacier page and click Inspect. Move to Sources tab, click top, click Glacier tax 1099-B... , open inject.js under src/inject and set a breakpoint by clicking on the line136. Choose file again and see what is going on there. "

Please let me know what you find!

**Comment by anmolnehru** [comment]
Hi just wanted to update with what I found. It actually errors out in line 25, thinking filetype isn't matching for some reason.
     

            var file = fileInput.files[0];  ->file = File {name: "csv_robinhood_anmol.csv", lastModified: ..*,
            var textType = /text.csv/;   ->textType = /text.csv/ {lastIndex: 0}

            if (file.type.match(textType)) {    -->fails here for some reason
                var reader = new FileReader();

                reader.onload = function(e) {
                    var lines = processData(reader.result);
                    if (!lines) {
                        fileInput.value = "";


Could you look into this.

Again the assert is being hit from line 46 and not 125.

I am a fellow badger as well :). Thanks for the great work.

**Comment by JiahaoShan** [comment]
Yeahh! Just realized that. It's nice to see it helped someone around me. 
I tried to export csv using the tool again, import it and it works just fine. 
If the match fails, most likely it means that the file type you imported is not text.csv 
May I know what is file.type at this point?

**Comment by anmolnehru** [comment]
It is a .csv file. I opened it to make sure as well. Here is the first row.

name,acquired,sold,proceeds,cost


*Thanks & Regards,*

Anmol Mohanty,

UW-Madison CS MS (2016)
MyWebsite <http://pages.cs.wisc.edu/~amohanty/>

**Comment by anmolnehru** [comment]
Not sure why it's asserting thatwhen file type is clearly .csv.

Does it need to be of a specific format like unicode/endian/utf-8?



*Thanks & Regards,*

Anmol Mohanty,

UW-Madison CS MS (2016)
MyWebsite <http://pages.cs.wisc.edu/~amohanty/>

**Comment by anmolnehru** [comment]
Have you tested in on Windows platforms? Your screenshots show a mac. Not sure if that has some role to play here. I will attach a masked csv here that I using which you may try out at your end.
[github robinhood.zip](https://github.com/JiahaoShan/Glacier-tax-1099-B-Stock-Transactions-Helper/files/894808/github.robinhood.zip)

**Comment by JiahaoShan** [comment]
This is great! I just figured out that under different platforms or even browsers, the file type can be significantly different. For example, csv file type can be "text/comma-separated-values, text/csv, application/csv, application/excel, application/vnd.ms-excel or application/vnd.msexcel"

I have updated the extension and re-uploaded it to the chrome extension store. The change should be effective within an hour (new version is 0.2.2). 

Thanks again for this interesting issue.
